handlers-converters/handlers/document-worker/src/main/java/com/github/cafdataprocessing/worker/policy/handlers/documentworker/DocumentWorkerHandler.java

```diff
@@ -261,4 +267,22 @@ private static String parseCustomDataValue(final Object value, final TaskData ta
         LOG.warn("The customData object 'source' is not recognised. Skipping...");
         return null;
     }
+
+    private static String encodeInlineJson(final Map value, final ObjectMapper mapper)
+    {
+        // Get the 'data' object from the supplied value and check that it is a Map.
+        final Object data = value.get("data");
+        if (!(data instanceof Map)) {
```

Comment:
I don't think we should include this check.  We could just allow `data` to be any type of JSON Value, rather than restricting it to JSON Objects.  For example, there might be cases where you want to pass an array.